sector-builder/src/scheduler.rs

```diff
@@ -40,7 +42,10 @@ pub enum Request {
         SecondsSinceEpoch,
         mpsc::SyncSender<Result<SectorId>>,
     ),
-    GetSealedSectors(mpsc::SyncSender<Result<Vec<SealedSectorMetadata>>>),
+    GetSealedSectors(
+        bool, // check health
```

Comment:
can we make this an enum with a single filed, sth like Option::CheckHealth, that would be clearer

Reply:
done!